addons/mail/static/tests/new/discuss/sidebar_tests.js

```diff
@@ -441,3 +441,243 @@ QUnit.test("mark channel as seen on last message visible", async function (asser
         "o-unread"
     );
 });
+
+QUnit.test(
+    "channel - counter: should not have a counter if the category is unfolded and without needaction messages",
+    async function (assert) {
+        const pyEnv = await startServer();
+        pyEnv["res.users.settings"].create({
+            user_id: pyEnv.currentUserId,
+            is_discuss_sidebar_category_channel_open: true,
+        });
```

Comment:
Why is this extra setup needed compared to previous tests?
It feels it shouldn't be necessary for this test to work.

Reply:
The previous tests were relying on default values, which was unclear for me.
For example, I did not understand why the chat tests had to click on title.

To make the test more explicit, I prefer to pass that data and no longer rely on extra `click`.